rubric/rubric.json

```diff
@@ -1,80 +1,94 @@
+      "Expert": "I exemplify a supportive and productive feedback culture and embrace feedback and apply it when necessary. I can quickly spot bugs by just reviewing code and am proactively looking at ways to simplify a solution while adhering to the system's architecture.  While critical during the review process, I strive to create a culture of continuous learning and improvement."
     },
-    "Requirements": {
-      "Beginner": "Requires very detailed step-by-step overviews of requirements. Can carry very small tasks. Cannot estimate time requirements.",
-      "Advanced Beginner": "\"Can implement requirements but may be \"\"to the letter\"\" of acceptance criteria and no more. Will be familiar with just a few areas of a single application. Low quality estimation.\"",
-      "Intermediate": "\"Implements requirements both to the letter and spirit of acceptance criteria. Asks good questions during requirements grooming. However they may fall into \"\"Bikeshedding\"\" patterns or require very detailed specifications. Mixed estimation accuracy.\"",
-      "Advanced": "Frequently provides the organization / project with simpler optimized requirement alternatives. Can communicate organization / project needs to developers and design constraints to users. Can break down large requirements by domain or stack verticals. Good estimation accuracy. Effective at “pushing back” on the Business.",
-      "Expert": "Consistently anticipates the organization’s / project's needs and improves grooming to suit it. Capable of building roadmaps that balance technical efficiency with maximum value. Influences testing strategies to ensure quality. Excellent estimate accuracy. Highly effective at “pushing back” on the Business.\r"
+    "Understanding Requirements": {
+      "Beginner": "I am still learning the system and require very detailed step-by-step overviews of requirements along with acceptance criteria.",
+      "Advanced Beginner": "I am able to take requirements and fill in some technical gaps but still require defined acceptance criteria.",
+      "Intermediate": "I can implement requirements both to the letter and spirit of acceptance criteria as well as fill in gaps as necessary. I ask good questions during requirements grooming, however I sometimes fail to identify the real problem and can be taken down rabbit trails.",
+      "Advanced": "I frequently provide the project with simpler optimized requirement alternatives. I am comfortable communicating project needs to developers and design constraints to users. I am able to effectively break down large requirements by domain.",
+      "Expert": "I am able to anticipate the organization’s needs and can improve grooming to suit it. I'm capable of building roadmaps that balance technical efficiency with maximum value."
```

Comment:
```suggestion
      "Expert": "I am able to anticipate the organization’s needs and can improve grooming to suit it. I'm capable of building roadmaps that balance technical efficiency with maximum value. Highly effective at “pushing back” on the Business when conflicts come up."
```

Reply:
This captures an important element of being able to assess a proposal and logically discuss why that proposal may not be feasible so that stakeholders are able to propose alternative paths without a lot of time wasted on hurt-feelings.